frontend/src/components/eMIB/EditActionDialog.jsx

```diff
@@ -72,10 +72,12 @@ class EditActionDialog extends Component {
   };
 
   state = {
-    action: {}
+    action: {},
+    saveDisabled: false
```

Comment:
can we call this `isSaving` - this will be useful when we setup APIs.

Reply:
renamed in c0037d1